packages/video_player/video_player_android/pigeons/messages.dart

```diff
@@ -43,11 +43,12 @@ class PositionMessage {
 }
 
 class CreateMessage {
+  CreateMessage({required this.httpHeaders});
```

Comment:
Why is this required, rather than defaulting to an empty collection?

Reply:
Pigeon demands there be no logic here, it doesn't generate code for initialization.